aiida_siesta/workflows/utils/future_iterate.py

```diff
@@ -0,0 +1,205 @@
+from aiida.plugins import DataFactory
+from aiida.orm import KpointsData
+
+from ...calculations.tkdict import FDFDict
+from ..base import SiestaBaseWorkChain
+from .iterate_absclass import ProcessInputsIterator, GeneralIterator
+
+
+class SiestaBaseWorkChainInputsIterator(ProcessInputsIterator):
+    """
+    Iterator for the SietaBaseWorkChain. No parameters other than the SietaBaseWorkChain inputs
+    are allowed as keys of the input `iterate_over`.
+    """
+    _process_class = SiestaBaseWorkChain
+    _expose_inputs_kwargs = {'exclude': ('metadata',)}
+
+
+class SiestaBaseWorkChainInputsIterator2(ProcessInputsIterator):
+    """
+    Iterator for the SietaBaseWorkChain. No parameters other than the SietaBaseWorkChain inputs
+    are allowed as keys of the input `iterate_over`. Add namespace
+    """
+    _process_class = SiestaBaseWorkChain
+    _expose_inputs_kwargs = {'exclude': ('metadata',), "namespace": 'sies'}
```

Comment:
Is "sies" intentional? :)

Reply:
This is more a test WorkChain, to test the situation when you use namespaces. I don't think I will leave it at the end. I couldn't bother to write "siesta" in full! Ahah